dual_quaternions/setup.py

```diff
@@ -1,18 +1,18 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 def readme():
     with open('README.rst') as f:
         return f.read()
 
-setup(name='dual_quaternions_ros',
-      version='0.2.4',
-      description='Dual quaternion implementation for use with ROS',
+setup(name='dual_quaternions',
+      version='0.3.0',
```

Comment:
doesn't need to happen now, but should eventually say what the versioning scheme is in the readme

Reply:
I'll need to look into versioning schemes. I'm just increasing the smallest version number for incremental updates and the second one for larger or breaking changes. I'm reserving the first number for major releases, with the `0.x.x` indicating it's not long-term stable